packages/lexical-history/src/__tests__/unit/LexicalHistory.test.tsx

```diff
@@ -34,12 +34,12 @@ import {
 } from 'lexical/src';
 import {createTestEditor, TestComposer} from 'lexical/src/__tests__/utils';
 import React from 'react';
-import {createRoot} from 'react-dom/client';
+import {createRoot, Root} from 'react-dom/client';
```

Comment:
nit: we could import Root as type instead
```suggestion
import type {Root} from 'react-dom/client';
import {createRoot} from 'react-dom/client';
```